CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ❤️ **Donate:** Enjoying MagicMirror²? [Please consider a donation!](https://magicmirror.builders/#donate) With your help we can continue to improve the MagicMirror².
 
-## [2.30.0] - Unreleased (`develop` branch)
+## [2.30.0] - 25-01-01
```

Comment:
one little thing, can you use 2025-01-01 here as in the releases before, thx.

Reply:
sorry, typo.. had made the change in my test env, then redid for test in clean.. WHILE test was running, taking focus away from editor.. thx